types/requirejs/README.md

```diff
@@ -18,7 +16,7 @@ export classes so they can be accessed from an import:
 // File main.ts
 export class Main  {
 
-	run() {  
+	run() {
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/requirejs/README.md;
  Line: 19;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.